site/index.php

```diff
@@ -0,0 +1,43 @@
+<?php
+include('config/init.php');
+function add($param1, $param2){
+  return $param1 + $param2;
+}
+function subtract($param1, $param2){
+  return $param1 - $param2;
+}
+function multiply($param1, $param2){
+  return $param1 * $param2;
+}
+function divide($param1, $param2){
+  return $param1 / $param2;
+}
+
+echo "
+  <form action='' method='get'>
+    <input name='param1' /> <br>
+    <input type='radio' name='add' /> add <br>
+    <input type='radio' name='subtract' /> subtract <br>
```

Comment:
I think the idea with radio buttons is that they all have the same name, but they have different values. With how it's written now, I think you'd be able to select all of these different operators.